feature-libs/product-configurator/rulebased/cpq/cpq-configurator-rest.service.spec.ts

```diff
@@ -62,6 +122,11 @@ describe('CpqConfiguratorRestService', () => {
       providers: [
         CpqConfiguratorRestAdapter,
         { provide: OccEndpointsService, useClass: MockOccEndpointsService },
+        {
+          provide: CPQ_CONFIGURATOR_OVERVIEW_NORMALIZER,
+          useClass: DummyOvConverter,
+          multi: true,
```

Comment:
Not needed here I guess

Reply:
yes - as i now know how to test protected :-)